### PR TITLE
Assign 2

### PR DESCRIPTION
**Pull request by PREM3D2**
my assignment-2 has successfully ran testcases

### CODE REVIEW

**Comment by BrijneetaGupta** [comment]
Please add the screenshot of your html page.

File: src/Assignment-2/assignment2.css [hunk 1]
```diff
@@ -8,60 +8,82 @@ height: 100%;
...
 }
 
-img{
+.page-logo{
     width: 200px;
     height: 200px;
     padding-top:  20px;
     padding-bottom:10px;
 }
-form{
```
Comment (BrijneetaGupta) [inline src/Assignment-2/assignment2.css]: Didn't closed this class.

File: src/Assignment-2/assignment2.css [hunk 2]
```diff
@@ -8,60 +8,82 @@ height: 100%;
...
+    border-bottom: 1px solid black;
+}
+.page-header{
     border: 1px solid black;
     background:chartreuse;
     margin:10px;
+    height:230px;
 }
 
-img{
```
Comment (BrijneetaGupta) [inline src/Assignment-2/assignment2.css]: Remove this.

File: src/Assignment-2/assignment2.css [hunk 3]
```diff
@@ -8,60 +8,82 @@ height: 100%;
...
     border:1px solid black;
-    font-size: 40px;
+    background-color: coral;
     margin:10px;
-    padding:5px;
-    text-align:center;
-
+    text-align: center;
 }
-.mainbody{
```
Comment (BrijneetaGupta) [inline src/Assignment-2/assignment2.css]: To give two same styling to more than 1 class, we do it like below:
.class1name, .class2name{

}

File: src/Assignment-2/assignment2.css [hunk 4]
```diff
@@ -8,60 +8,82 @@ height: 100%;
...
+.page-body{
     width:100%;
+    margin:10px;
 }
-aside{
```
Comment (BrijneetaGupta) [inline src/Assignment-2/assignment2.css]: close this

File: src/Assignment-2/assignment2.css [hunk 5]
```diff
@@ -8,60 +8,82 @@ height: 100%;
...
-    width:20%;
-    background:greenyellow;
-    border:1px solid black;
-    
+aside,section{
+    display: inline-block;
 }
-.left-sidebar ul li{
```
Comment (BrijneetaGupta) [inline src/Assignment-2/assignment2.css]: close this

**Review by BrijneetaGupta — COMMENTED** [review]
Go through your .css file again. At many places you missed closing the tag.

**Comment by PREM3D2** [comment]
i think i have closed all the tags in css . In every mistake you mentioned
, i have closed the tags but a line below please check once . Also, i'am
adding my UI screen shots for your reference.

**Review by BrijneetaGupta — APPROVED** [review]

**Comment by SuneetKumar19** [comment]
Good work

**Pull request merged**